content/en/docs/ops/configuration/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Understanding Traffic Routing
+linktitle: Traffic Routing
+description: How Istio routes traffic through the mesh.
+weight: 30
+keywords: [traffic-management,proxy]
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+One of the goals of Istio is to act as a "transparent proxy" that can be dropped in place into an existing cluster and allow traffic to continue to function as previously.
+However, there are some important ways that Istio causes traffic to be handled differently than a typical Kubernetes cluster, due to additional features (such as request load balancing) and architectural constraints.
+To understand what is happening in your mesh, it is important to understand how Istio routes traffic.
+
+{{< warning >}}
+This document describes low level implementation details. For a higher level overview, check out the traffic management [Concepts](/docs/concepts/traffic-management/) or [Tasks](/docs/tasks/traffic-management/).
+{{< /warning >}}
+
+## Protocols
+
+Unlike Kubernetes, Istio has the ability to process higher level protocols such as HTTP and TLS.
+In general, there are three classes of protocols Istio understands:
+
+* HTTP, which includes HTTP/1.1, HTTP/2, and gRPC. Note that this does not include TLS encrypted traffic (HTTPS).
+* TLS, which includes HTTPS.
+* Opaque TCP bytes.
+
+Which protocol is used is determined by [protocol selection](/docs/ops/configuration/traffic-management/protocol-selection/).
+
+The use of "TCP" can be confusing, as in other contexts it is used to distinguish between other L4 protocols, such as UDP.
+When referring to the TCP protocol in Istio, this typically means we are treating it as an opaque stream of bytes,
+and not parsing higher level constructs such as TLS or HTTP.
+
+## Traffic Routing
+
+When the Istio proxy receives a request, it must decide where to forward the request to, if anywhere.
+By default, this will be the original service that was request, unless [customized](/docs/tasks/traffic-management/traffic-shifting/).
+How this works depends on the protocol used.
+
+### TCP
+
+When processing TCP traffic, Istio has a very small amount of useful information to route the connection - only the destination IP and Port.
+Fortunately, this is enough to determine the intended Service; the proxy is configured to listen on each service IP:Port pair and forward traffic to that service.
+
+For customizations, a TCP `VirtualService` can be configured, which allows [matching on specific IPs and ports](/docs/reference/config/networking/virtual-service/#L4MatchAttributes).
+
+### TLS
+
+When processing TLS traffic, Istio has slightly more information available than raw TCP: we can inspect the [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication).
```

Comment:
SNI field?  SNI header?

Reply:
Based on the wiki article it seems this is how its used, although it does sound odd

Reply:
But neeraj said it as well so I will go with it